### ISSUE TITLE
New models trained yolo8

### ISSUE THREAD

**Issue by maurobotta** [DonkeySmall/Delphi-Tensorflow-Lite#5]
Hi, i have created a new Yolo8 model ( first time for me) ,

model = YOLO("yolov8n.yaml")
result = model.train(data="config.yaml", epochs=100)
model.export(format="tflite")

Now  i have these files : best_float16.tflite , best_float32.tflite, 

i have these files try with Delphi Tensor library, but don't work. ( no error ) 

Hint ?

info:

Ultralytics YOLOv8.0.228 🚀 Python-3.11.5 torch-2.1.2+cpu CPU (AMD Ryzen 7 5800X 8-Core Processor)
engine\trainer: task=detect, mode=train, model=yolov8n.yaml, data=coco128.yaml, epochs=100, time=None, patience=50, batch=16, imgsz=640, save=True, save_period=-1, cache=False, device=None, workers=8, project=None, name=train15, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=None, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, show=False, save_frames=False, save_txt=False, save_conf=False, save_crop=False, show_labels=True, show_conf=True, show_boxes=True, line_width=None, format=torchscript, keras=False, optimize=False, int8=False, dynamic=False, simplify=False, opset=None, workspace=4, nms=False, lr0=0.01, lrf=0.01, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=7.5, cls=0.5, dfl=1.5, pose=12.0, kobj=1.0, label_smoothing=0.0, nbs=64, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=0.0, translate=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.5, mosaic=1.0, mixup=0.0, copy_paste=0.0, cfg=None, tracker=botsort.yaml, save_dir=runs\detect\train15

--

Ultralytics YOLOv8.0.228 🚀 Python-3.11.5 torch-2.1.2+cpu CPU (AMD Ryzen 7 5800X 8-Core Processor)
YOLOv8n summary (fused): 168 layers, 3151904 parameters, 0 gradients, 8.7 GFLOPs

PyTorch: starting from 'runs\detect\train15\weights\best.pt' with input shape (1, 3, 640, 640) BCHW and output shape(s) (1, 84, 8400) (6.2 MB)

TensorFlow SavedModel: starting export with tensorflow 2.15.0...
WARNING ⚠️ tensorflow<=2.13.1 is required, but tensorflow==2.15.0 is currently installed https://github.com/ultralytics/ultralytics/issues/5161

ONNX: starting export with onnx 1.15.0 opset 17...
ONNX: simplifying with onnxsim 0.4.35...
ONNX: export success ✅ 1.2s, saved as 'runs\detect\train15\weights\best.onnx' (12.2 MB)
TensorFlow SavedModel: running 'onnx2tf -i "runs\detect\train15\weights\best.onnx" -o "runs\detect\train15\weights\best_saved_model" -nuo --non_verbose'
WARNING:tensorflow:From C:\xxxxxxxx\AppData\Local\Programs\Python\Python311\Lib\site-packages\keras\src\losses.py:2976: The name tf.losses.sparse_softmax_cross_entropy is deprecated. Please use tf.compat.v1.losses.sparse_softmax_cross_entropy instead.

TensorFlow SavedModel: export success ✅ 20.1s, saved as 'runs\detect\train15\weights\best_saved_model' (30.6 MB)

TensorFlow Lite: starting export with tensorflow 2.15.0...
TensorFlow Lite: export success ✅ 0.0s, saved as 'runs\detect\train15\weights\best_saved_model\best_float32.tflite' (12.2 MB)

Export complete (23.8s)
Results saved to V:\Python\CORSO\YOLO\runs\detect\train15\weights
Predict:         yolo predict task=detect model=runs\detect\train15\weights\best_saved_model\best_float32.tflite imgsz=640
Validate:        yolo val task=detect model=runs\detect\train15\weights\best_saved_model\best_float32.tflite imgsz=640 data=C:\xxxxxxx\AppData\Local\Programs\Python\Python311\Lib\site-packages\ultralytics\cfg\datasets\coco128.yaml
Visualize:       https://netron.app

**Comment by DonkeySmall** [comment]
Send me your tflite model, [donkey.small@yandex.ru](mailto:donkey.small@yandex.ru)

**Comment by maurobotta** [comment]
> [donkey.small@yandex.ru](mailto:donkey.small@yandex.ru)

do now
